src/halo-hooks/useTVLInfo.ts

```diff
@@ -49,11 +53,22 @@ const useTVLInfo = () => {
     })
   }
 
-  function setData(data: any) {
+  async function setData(data: any) {
+    const uri = 'token_price/ethereum?contract_addresses='
+    const concatString = HALO_TOKEN_ADDRESS[ChainId.MAINNET]
+    let usdPrice = 0
+    const url = `${COINGECKO_API_URL}/simple/${uri}${concatString}&vs_currencies=usd`
```

Comment:
@geonellov are we able to re-use `getTokensUSDPrice()` function here instead of writing the cg api call again?

Reply:
@schystz done re-using `getTokensUSDPrice()`